FormlyMapper/FormlyField/DateField.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+use Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+class DateField extends FormlyField
+{
+    /**
+     * @return array
+     */
+    public function getFieldConfiguration()
+    {
+        $this->formlyFieldConfiguration['templateOptions']['type'] = 'date';
```

Comment:
Getters should not be changing state. These changes should be done in another method. I would probably have a method in the abstract class that returns the `formlyFieldConfiguration`, and have that method call an abstract method that that makes the changes.

You could even have an abstract method that simply returns the type, and an optional method that performs any other configuration as necessary.